platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -88,14 +88,16 @@ Enterprises adopting platform engineering can expect to achieve the following:
 
 1. Accelerate product development and delivery by intentionally dedicating teams
    to platform services and thereby reducing cognitive load on product teams
-1. Improve reliability and resiliency of products relying on platform
-   capabilities by dedicating experts to configure and manage them
-1. Accelerate product development and delivery by reusing and sharing platform
+2. Improve reliability and resiliency of products relying on platform
+   capabilities by dedicating experts to develop, configure and manage them
```

Comment:
We didn't include "develop" because in some platform teams they don't actually construct the services and capabilities, rather they manage those provided by a cloud provider.

Reply:
I would argue that even the tailoring and configuration of cloud services is some sort of development. It it for sure developing a developer platform and I doesn't require writing actual code.